test/core/handshake/client_ssl.cc

```diff
@@ -175,11 +173,10 @@ static int alpn_select_cb(SSL* /*ssl*/, const uint8_t** out, uint8_t* out_len,
 
 static void ssl_log_where_info(const SSL* ssl, int where, int flag,
                                const char* msg) {
-  if ((where & flag) &&
-      GRPC_TRACE_FLAG_ENABLED(client_ssl_tsi_tracing_enabled)) {
-    LOG(INFO) << absl::StrFormat("%20.20s - %30.30s  - %5.10s", msg,
-                                 SSL_state_string_long(ssl),
-                                 SSL_state_string(ssl));
+  if (where & flag) {
```

Comment:
I'd prefer `(where & flag) != 0` here, for the same reasons we usually write `!= nullptr` for pointers -- this is an int not a bool.

Reply:
+1 -- but don't need the parens.

Reply:
Fixed. It does require parens though, `!=` has higher precedence than `&`. http://go/cppref/cpp/language/operator_precedence